content/jobs/2022/open-source-infrastructure-engineer.md

```diff
@@ -1,16 +1,24 @@
 +++
 authors =  ["Damián Avila"]
-title = "Open Source Infrastructure Engineer"
-work_description = ["Cloud infrastructure management and operations.", "Development of open source infrastructure for hosted JupyterHub service"]
+title = "Open Source Infrastructure Engineer: Site Reliability Engineering and Cloud Infrastructure"
+work_description = [
+  "Site Reliability Engineering",
+  "Cloud infrastructure management, operations, and support.",
+  "Development of open source infrastructure for hosted JupyterHub service"
+]
 salary_range = "$120,000 - $135,000"
 location = "preferred at the US/Pacific time zone"
 date = "2022-10-14"
 url_apply = "https://forms.gle/RPqWHoweDnvUVp4d8"
-deadline = "We will begin reviewing applications around **Oct 17th**, and will accept them on a rolling basis until the position is filled."
+deadline = "We will begin reviewing applications around **November 17th**, and will accept them on a rolling basis until the position is filled."
```

Comment:
Do you think it is OK to update the date (thinking of the ones who already applied)?
Probably OK but just checking with you...

Reply:
I think it's ok, I think of it as a reflection of reality more than anything, since we haven't actually started the process yet. I feel like it's best to be honest with folks. We could always add an edit note saying that we changed the date. Maybe that's a nice compromise?

Reply:
OK to me.